code/datums/uplink_item.dm

```diff
@@ -239,33 +239,45 @@ var/list/uplink_items = list()
 /datum/uplink_item/dangerous/gatling
 	name = "Gatling Gun"
 	desc = "A huge minigun. Makes up for its lack of mobility and discretion with sheer firepower. Has 200 bullets."
-	item = /obj/item/weapon/gun/gatling
+	item = /obj/structure/closet/crate/secure/weapon/experimental/gatling
+	cost = 40
+	jobs_exclusive = list("Nuclear Operative")
+
+/datum/uplink_item/dangerous/nikita
```

Comment:
Every time someone uses this they end in nullspace due the nikita being sheer shitcode, i'd avoid it.

Reply:
I've not had this problem myself when testing the crates/loadouts, is it truly still an ongoing issue ?